### PR TITLE
chore:update pricing plans

### PR DESCRIPTION
**Pull request by adu-yeboah**

### CODE REVIEW

File: apps/client/app/modules/landing-page/components/pricing/index.tsx [hunk 1]
```diff
@@ -3,50 +3,54 @@ import {CheckIcon} from '@heroicons/react/24/outline'
 
 const tiers = [
   {
-    name: 'Silver Snapshot Plan',
+    name: 'Snap & Share',
     id: 'tier-hobby',
-    href: '#',
+    href: '/auth?pricing_package=FREE',
     priceMonthly: '$9.99',
```
Comment (arkhurst) [inline apps/client/app/modules/landing-page/components/pricing/index.tsx:9]: I believe we put the correct pricing there. can we update where neccessary

**Review by arkhurst — CHANGES_REQUESTED** [review]

**Comment by adu-yeboah** [comment]
the price was not provided since it wasn't part of the task

**Review by Bendomey — APPROVED** [review]

**Review by arkhurst — APPROVED** [review]

**Pull request merged**